util/tc.py

```diff
@@ -48,18 +50,24 @@ def maybe_download_tc_bin(**kwargs):
 if __name__ == '__main__':
     if len(sys.argv) < 2:
         print('Downloads and extracts native client binaries.')
-        print('Usage: {} <target_dir> [arch_string]'.format(sys.argv[0]))
+        print('Usage: {} <target_dir> [arch_string] [artifact_name]'.format(sys.argv[0]))
         print('  <target_dir>: Where to put the native client binary files')
-        print('  [arch_string]: Which architecture to download binaries for. "arm" for ARM 7 (32-bit), "gpu" for CUDA enabled x86_64 binaries, "cpu" for CPU-only x86_64 binaries. Optional ("cpu" by default)')
+        print('  [arch_string]: Which architecture to download binaries for. "arm" for ARM 7 (32-bit), "gpu" for CUDA enabled x86_64 binaries, "cpu" for CPU-only x86_64 binaries, "osx" for CPU-only x86_64 OSX binaries. Optional ("cpu" by default)')
+        print('  [artifact_name]: Name of the artifact to download. Defaults to "native_client.tar.xz"')
         exit(1)
 
-    if len(sys.argv) == 2:
-        arch_string = 'cpu'
-    else:
-        arch_string = sys.argv[2]
+    arch_string   = 'cpu'
+    artifact_name = 'native_client.tar.xz'
+
+    if len(sys.argv) == 3:
```

Comment:
Maybe time to add argparse?

Reply:
I won't block on it, but probably a good idea.